README.md

```diff
@@ -34,9 +34,20 @@ Examples:
 ## Requirements
 
 - [Pandoc](https://pandoc.org/installing.html)
+  - Ubuntu:
+    - `sudo apt install pandoc`
 - LaTeX (eg. [TeX Live](http://www.tug.org/texlive/)) in order to get `pdflatex` or `xelatex`
+  - Ubuntu:
+    - `sudo apt install texlive texlive-xetex`
+    - `tlmgr option repository ftp://tug.org/historic/systems/texlive/2019/tlnet-final`
```

Comment:
It' not mandatory to use `tlmgr`, you could have installed `texlive-latex-extra` or `texlive-full` in order to avoid to download extensions manually.

Reply:
Okay, then we remove those lines.